src/dapp/components/farm/Farm.tsx

```diff
@@ -69,6 +71,14 @@ export const Farm: React.FC = () => {
     BlockchainState
   >(service);
 
+  const [showModal, setShowModal] = React.useState(false);
```

Comment:
We have got a few modals in the game, consider calling this something a bit more descriptive.

E.g. `showExploreModal`